src/api/public/apidocs-new/paths/source_project_name_cmd_unlock.yaml

```
@@ -0,0 +1,52 @@
+post:
+  summary: Unlocks the project
```

```suggestion
  summary: Unlocks the project.
```
Missing dot at the end of the sentence.

---

done